apps/frontend/src/app/(app)/profile/page.tsx

```diff
@@ -42,6 +58,19 @@ const ProfilePage: React.FC = () => {
           <h1 className="mt-8 text-xl font-medium text-white">
             Personal Integrations
           </h1>
+          <div className="flex flex-col items-start space-y-2">
+            <h2 className="text-lg font-medium text-white">GitHub Integration</h2>
+            {isGithubConnected ? (
+              <p className="text-green-400">GitHub account connected successfully!</p>
+            ) : (
+              <button
+                onClick={handleConnectGithub}
+                className="mt-2 bg-blue-600 text-white py-2 px-4 rounded"
+              >
+                Connect GitHub
+              </button>
+            )}
+          </div>
```

Comment:
I don't think any of this is required! We already have the buttons in place

![image](https://github.com/user-attachments/assets/70abaaf9-c82c-40dc-85ba-7b767660c04b)

Reply:
which app it is @madhav-relish 😿